core/lib/mcxtx.h

```diff
@@ -229,7 +229,8 @@
          * Windows system calls.  These fields are ignored for 32-bit processes
```

Comment:
Should this be updated for i#3422 where ymmh needs to be saved: I don't remember what the conclusion there is -- are we going to just be saving all these xmm+ymm regs all the time?

Reply:
You mean adding xref? I wasn't planning to fix the Windows case at the beginning.

Reply:
OK delay until fixed.